etc/helm/pachyderm/values.yaml

```diff
@@ -14,7 +14,7 @@ global:
     # If blank, a value will be generated by the postgres subchart
```

Comment:
Looks like an auto formatter got run on this file?

Reply:
autoformatter is my arch nemesis 😡